sample/tile.yml

```diff
@@ -10,6 +10,7 @@ icon_file: resources/icon.png
 # docker_bosh_version: '28'           # Optional, default: latest release
 # apply_open_security_group: true     # Apply open security group, default: false
 allow_paid_service_plans: true        # Allow paid service plans, default: false
+stanalone: false                      # Include default CF properties in property_blueprints metadata, default: false
```

Comment:
This is a little misleading. The comment suggests that the option "Includes default CF properties..", but it is defaulted to off.

Reply:
Oops, that's backwards. Should be "Suppress"